website/docs/cloud-docs/users-teams-organizations/organizations.mdx

```diff
@@ -212,6 +212,24 @@ Organization owners can set up an SSO provider for the organization.
 
 #### General
 
+Organization owners can choose between Non-Aggregated (Default) and Aggregated status checks. This setting determines whether detailed information and links are accessed directly from the VCS Provider or Terraform Cloud.
+It also determines the quantity of status checks directly sent to the VCS Provider in response to actions such as Pull/Merge requests.
+
+##### Aggregated Status Checks
+
+Aggregated status checks offer a streamlined experience to customers with "monorepo" repositories, containing configuration for a large number of workspaces.
```

Comment:
In the docs we like to talk directly to the user, so we avoid using terms like "customer" in favor of directly using "you":
```suggestion
Aggregated status checks offer a streamlined experience if you have a single repository containing configuration for many workspaces (a.k.a., a monorepo).
```